Examples/Aurora/affinity_gpu/sycl/main.cpp

```diff
@@ -134,11 +80,29 @@ int main(int argc, char *argv[])
         char list_cores[7*CPU_SETSIZE];
         get_cores(list_cores);
 
-        printf("To affinity and beyond!! nname= %s  rnk= %d  list_cores= (%s)  num_devices= %i  gpu_uuid= ",
+        printf("To affinity and beyond!! nname= %s  rnk= %d  list_cores= (%s)  num_devices= %i  gpu_uuids= ",
```

Comment:
Nitpicking but I guess we should not mitch and match `prinf` and `stdcout`

Reply:
You're right! I'll change it in a bit!